go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/gardener/gardener-extension-provider-openstack
 
-go 1.13
+go 1.14
```

Comment:
Just a reminder: We also have to update the `meta/ci` ref and the pipeline definition there after this PR is merged. Can you take care? :)

Reply:
Absolutely! Thanks for letting me know.